src/pipeline.py

```diff
@@ -1,10 +1,17 @@
-from dplypy import DplyFrame
+from src.dplypy import DplyFrame
 
-def query(query_str: str): # summary_type: e.g. mean, variance, other descriptive stats, num rows, 
+
+def query(query_str: str):  # summary_type: e.g. mean, variance, other descriptive stats, num rows,
     return lambda d1: DplyFrame(d1.pandas_df.query(query_str))
 
-def apply(func, axis=0, **kwargs): # TODO: CSC: use kwargs... couldn't quickly figure out how to do it
-    return lambda d1: DplyFrame(d1.pandas_df.apply(func=func, axis=axis)) 
+
+def apply(func, axis=0, **kwargs):  # TODO: CSC: use kwargs... couldn't quickly figure out how to do it
+    return lambda d1: DplyFrame(d1.pandas_df.apply(func=func, axis=axis))
+
+
+def drop(labels=None, axis=0, index=None, columns=None):
```

Comment:
I know this hasn't been done yet, but let's get in the habit writing API documentation for the function as it's implemented.

Reply:
Got it! Updated in my next commit. Do we need to write documentation for tests as well?